### ISSUE TITLE
Rework cifar10 distributed tutorial

### ISSUE THREAD

**Issue by vfdev-5** [pytorch-ignite/examples#67]
The idea is to restructure this tutorial in the following way:
We can provide 2 parts:
1) minimal DDP code with 1 minimalistic trainer and 1 evaluator (computing accuracy metric) = goal to put the most pertinent info in minimum lines of text
2) show how to add other features from current text: logging, resuming from checkpoint, etc.

@Priyansi @trsvchn what do you think ?

**Comment by Ishan-Kumar2** [comment]
Hi @vfdev-5 
I agree, we should point out the differences that need to be made to your code in order to convert a normal code to a distributed code using a minimal trainer and model. To do that should we have it as a how to guide which assumes you know how to write the basic Ignite pipeline and just mention the additions for distributed training?

**Comment by vfdev-5** [comment]
> To do that should we have it as a how to guide which assumes you know how to write the basic Ignite pipeline and just mention the additions for distributed training?

Yeah, maybe point (1) "minimal DDP code with 1 minimalistic trainer and 1 evaluator (computing accuracy metric) = goal to put the most pertinent info in minimum lines of text" could be a how-to guide
